nextflow/find_candidate_position.nf

```diff
@@ -343,10 +319,10 @@ process publish_best_pointing {
     publishDir params.final_dir, mode: 'copy'
 
     input:
-    file fits
+    path fits
 
     output:
-    path '*', includeInputs: true
+    path '*' includeInputs true
```

Comment:
@NickSwainston Can you just confirm that this formatting is correct? I had to make an educated guess to resolve a conflict.

Reply:
I just tried running that script and it did not give me any formatting errors so I believe it is correct